controllers/storagecluster/storageclient.go

```diff
@@ -2,19 +2,27 @@ package storagecluster
 
 import (
 	"fmt"
+	"strconv"
 
 	ocsclientv1a1 "github.com/red-hat-storage/ocs-client-operator/api/v1alpha1"
 	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v4/v1"
 	"github.com/red-hat-storage/ocs-operator/v4/controllers/util"
 
+	"golang.org/x/exp/maps"
```

Comment:
We are not using this library in the existing code, We are only using `maps`, pls change this.